src/origins.js

```diff
@@ -24,7 +24,7 @@ export default {
   },
   'atlassian.com': {
     url: '*://*.atlassian.com/*',
```

Comment:
There is also the 'atlassian.net' entry below 🤔 
Not sure if it makes sense to add both origins to the file header, or keep it like this. 🤷‍♂️

Reply:
Looks like the .net will redirect to .com